buildflow/runtime/runner.py

```diff
@@ -34,6 +41,10 @@ class _ProcessorRef:
     output_ref: type
 
 
+_SESSION_DIR = os.path.join(tempfile.gettempdir(), 'buildflow')
```

Comment:
maybe write this to ~/.config/buildflow

Reply:
thats where a lot of other frameworks dump stuff

Reply:
done.